backend/src/config.ts

```diff
@@ -24,6 +24,22 @@ export const DAPP_DID_URI = loadEnv('DAPP_DID_URI') as Kilt.DidUri
 export const DAPP_NAME = process.env.DAPP_NAME ?? 'Web3-Login-Demo'
 export const JWT_SIGNER_SECRET = loadEnv('JWT_SIGNER_SECRET')
 
+// To determine which CType to use, refer to the CType Hash
+const SOCIAL_KYC_EMAIL_CTYPE = 
+'0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac' 
+// Required properties for socialKYC  
+export const SOCIAL_KYC_EMAIL_REQUIRED_PORPERTIES = 'Email' 
```

Comment:
```suggestion
export const SOCIAL_KYC_EMAIL_REQUIRED_PROPERTIES = 'Email' 
```

Reply:
Also, why are these exported? Aren't they supposed to be consumed via the exported properties below, which also account for env values? These should probably not be exported then, as it is done for the `SOCIAL_KYC_EMAIL_CTYPE` value.